src/test/java/digital/loom/rhizome/authentication/AuthenticationTest.java

```diff
@@ -70,7 +70,7 @@
             clientId,
             "loom.auth0.com" );
     private static final AuthenticationAPIClient client = auth0.newAuthenticationAPIClient();
```

Comment:
We need a way to generate realistic auth0 questions. Not sure how to generate one that contains all the scope stuff.